fedimint-server/src/consensus/server.rs

```diff
@@ -264,7 +265,7 @@ impl ConsensusServer {
                 Ok(consensus_hash) if consensus_hash == our_hash => break,
                 Ok(_) => bail!("Our consensus config doesn't match peers!"),
                 Err(e) => {
-                    warn!(target: LOG_CONSENSUS, "ERROR {:?}", e)
+                    warn!(target: LOG_CONSENSUS, "Could not check consensus config hash: {}", OptStracktrace(e))
```

Comment:
@douglaz You showed me the `{:?}` printing stacktrace, but it seems to me now that oftentimes this creates tons of output for errors that are expected, so I created this `OptStacktrace` wrapper to have ability to get the stacktrace if really needed without being too noisy. We'll have to find some balance, case by case, I guess.